src/aluno/PerfilPage.tsx

```
@@ -0,0 +1,213 @@
+                                                <li key={projeto.id}>
+                                                    <button
+                                                        className="w-full text-left p-2 !border !bg-transparent !border-gray-200"
+                                                        onClick={() => {
+                                                            setSelectedProjeto(projeto)
+                                                            setDialogOpen(true)
+                                                        }}
+                                                    >
+                                                        <div className="flex justify-between items-center">
+                                                            <span className="font-medium">{projeto.titulo}</span>
+                                                            <span
+                                                                className={`px-3 py-1 rounded-full text-sm font-medium ${projeto.status === 'EM_ANDAMENTO'
```

ESLint

[error] Replace `··································className={`px-3·py-1·rounded-full·text-sm·font-medium·${` with `<span⏎································className={`px-3·py-1·rounded-full·text-sm·font-medium·${⏎··································`

(prettier/prettier)